pkg/trace/stats/quantile/converter.go

```diff
@@ -0,0 +1,171 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package quantile
+
+import (
+	"errors"
+	"sort"
+
+	"github.com/DataDog/sketches-go/ddsketch/mapping"
+	"github.com/DataDog/sketches-go/ddsketch/pb/sketchpb"
+	"github.com/golang/latestprotobuf/proto"
+)
+
+// ddSketch represents the sketch described here: http://www.vldb.org/pvldb/vol12/p2195-masson.pdf
+// This representation only supports positive values.
+type ddSketch struct {
+	// bins is the map from index --> count
+	bins map[int32]float64
+	// contiguousBins is a more compact representation for contiguous bins.
+	// the index of each bin is the index in the array + contiguousBinsOffset
+	contiguousBins       []float64
+	contiguousBinsOffset int
+	// zeros is the count of 0 and its close neighbors (close defined by gamma).
+	zeros int
+	// mapping is the mapping of the ddSketch: interpolation, gamma and global index offset.
```

Comment:
This is rocket science to me, but I trust that anyone reading the `pdf` linked above will know all of these terms so thanks for documenting 👍